static/js/src/cookie-policy-with-callback.js

```diff
@@ -0,0 +1,35 @@
+/*
+  Set a unique user_id if the user has accepted cookies.
+  If the user has not set their cookie preference, wait for the cookie policy to run first. 
+*/
+
+import { v4 as uuidv4 } from "https://jspm.dev/uuid";
+
+const getCookie = (targetCookie) =>
+  document.cookie.match(new RegExp("(^| )" + targetCookie + "=([^;]+)"));
+let cookieAcceptanceValue = getCookie("_cookies_accepted");
+
+if (cookieAcceptanceValue === null) {
```

Comment:
what if the value is None?

Reply:
The value of that cookie when not set is `null`, but updated to catch more falsey values